src/feature.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  EvaluationApi,
+  EvaluationBatchRequest,
+  EvaluationBatchRequestFlagTagsOperatorEnum,
+  EvaluationEntity,
+  createConfiguration,
+  ServerConfiguration,
+} from "flagr-client";
+
+export interface FlagCallbacks {
+  [key: string]: (...args: any) => Promise<any>;
+}
+
+export interface Variant {
+  key: string | null;
+  attachment: any | null;
+}
+
+const nullVariant: Variant = {
+  key: null,
+  attachment: null,
+};
+
+export interface Config {
+  flagrUrl: string;
+  tags?: string[];
+  tagOperator?: EvaluationBatchRequestFlagTagsOperatorEnum;
+}
+
+export function createFeature(config: Config) {
+  return new Feature(config);
+}
+
+export class Feature {
+  private _context: Object = {};
```

Comment:
**nitpick:** `Object` is as bad as `any`, since functions and arrays are also `Object`s. Could maybe use `Record<string, any>`?‏